ReactWindows/ReactNative.Shared/Bridge/CachedJavaScriptBundleLoader.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading.Tasks;
+using PCLStorage;
+
+namespace ReactNative.Bridge
+{
+    class CachedJavaScriptBundleLoader : JavaScriptBundleLoader
+    {
+        private readonly string _cachedFileLocation;
+        private string _script;
+
+        public CachedJavaScriptBundleLoader(string sourceUrl, string cachedFileLocation)
+        {
+            SourceUrl = sourceUrl;
+            _cachedFileLocation = cachedFileLocation;
+        }
+
+        public override string SourceUrl { get; }
+
+        public override async Task InitializeAsync()
+        {
+            try
+            {
+                var localFolder = FileSystem.Current.LocalStorage;
+                var storageFile = await localFolder.GetFileAsync(_cachedFileLocation).ConfigureAwait(false);
+                _script = storageFile.Path;
+            }
+            catch (Exception ex)
+            {
+                var exceptionMessage = FormattableString.Invariant($"File read exception for asset '{SourceUrl}'.");
+                throw new InvalidOperationException(exceptionMessage, ex);
+            }
```

Comment:
Since that was in the original code, can we handle it in a separate PR? At that point, we can also write a unit test that pins the exception message behavior.